modules/oauth-configuring-internal-oauth.adoc

```diff
@@ -3,21 +3,53 @@
 // * authentication/configuring-internal-oauth.adoc
 
 [id="oauth-configuring-internal-oauth_{context}"]
-= Configuring options for the internal OAuth server
+= Configuring the internal OAuth server's token duration
 
-You can configure default options for the internal OAuth server's 
+You can configure default options for the internal OAuth server's
 token duration.
 
+[IMPORTANT]
+====
+By default, tokens are only valid for 24 hours. Existing sessions
+will expire after this time.
+====
+
 .Procedure
 
-. Set the token duration options:
+. Define a configuration file that contains the token duration options:
 +
 [source,yaml]
 ----
-oauthConfig:
-  ...
+apiVersion: config.openshift.io/v1
+kind: OAuth
+metadata:
+  name: cluster
+spec:
   tokenConfig:
     accessTokenMaxAgeSeconds: 86400 <1>
 ----
 <1> Set `accessTokenMaxAgeSeconds` to control the lifetime of access tokens.
 The default lifetime is 24 hours, or 86400 seconds.
+
+. Apply the newly defined file.
++
+[NOTE]
+====
+The `oc apply` command must be used, as this updates the existing
+OAuth server.
+====
++
+----
+$ oc apply -f </path/to/file.yaml>
+----
+
+. Confirm that the changes have taken affect:
++
+----
+$ oc describe oauth.config.openshift.io/cluster
+...
+Spec:
+  Token Config:
+    Access Token Max Age Seconds:  86400
```

Comment:
Isn't that the default value? Consider using a different max age in both the config yaml and sample output.

Reply:
I've adjusted this to be twice the default value and confirmed this works.